Polygon_mesh_processing/doc/Polygon_mesh_processing/NamedParameters.txt

```diff
@@ -369,6 +369,13 @@ and make the operation impossible with the current version of the code.
 <b>Default:</b> `false`
 \cgalNPEnd
 
+\cgalNPBegin{do_not_modify} \anchor PMP_do_not_modify
+Parameter used in `corefine()` function to prevent the refinement of one of the two input meshes.
+\n
+<b>Type:</b> `bool` \n
+<b>Default:</b> `false`
```

Comment:
I find the semantic of `do_not_modify=false` or `do_not_modify=true` confusing. Could the name of the option be something with positive wordking, like `mutable_mesh` (with `true` as default), or `read_only_mesh`/`constant_mesh` (with `false` as default)? In my opinion that would be clearer.

Do other reviewers agree with that?

Reply:
I prefer the `do_not_...` because the name of the function implies that the function performs a refinement on both meshes. As such, this named parameter is an advanced option that makes it almost a different function (~ `refine()`), and the `do_not_...` reflects this better, in my opinion.